Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "typst-live"
+name = "typst-live-dev"
```

Comment:
forgot to un-rename?

Reply:
I had changed it, but forgot to push. Good spot, thanks!